app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,104 @@
+    <string name="action_default">Standart</string>
+    <string name="action_website_add">Webseite hinzufügen</string>
+    <string name="action_website_newwebsite">Neue Webseiten verfügbar!</string>
+    <string name="action_website_add_custom">Eigene Webseite hinzufügen</string>
+    <string name="copied">In Zwischenablage kopieren</string>
+    <string name="not_implemented" translatable="false">Not implemented</string>
+    <string name="action_website_noteditable_toast">Du kannst keine Remote Webseite editieren</string>
+
+    <string name="app_share_credits"> - via Quote http://bit.ly/anecdoteapp </string>
+
+    <string name="anecdote_share_title">Teilen via</string>
+    <string-array name="anecdote_content_dialog">
+        <item>Kopiere Link</item>
+    </string-array>
+
+
+    <string name="action_share">Teilen</string>
+    <string name="action_copy">Kopieren</string>
+    <string name="action_open">Öffnen</string>
+
+    <string name="open_intent_browser">Öffne Browser…</string>
+    <string name="open_intent_browser_error">Fehler beim Öffnen der Details, bitte URL checken</string>
+
+    <string name="action_website">Webseitenaktion</string>
+
+    <string name="dialog_website_new_title">Neue Webseite</string>
+    <string name="dialog_website_edit_title">Editieren</string>
+    <string name="dialog_website_add">Hinzufügen</string>
+    <string name="dialog_website_error_itemperpage">Die Anzahl der Objekte pro Webseite müssen mehr als 0 sein</string>
+    <string name="dialog_website_error_name">Möchtest du nicht einen Namen hinzufügen?</string>
+    <string name="dialog_website_error_pagename">Füge einen Namen hinzu</string>
+    <string name="dialog_website_error_url">Fehlende URL :(</string>
+    <string name="dialog_website_error_selector">Missing text selector</string>
+    <string name="dialog_website_save">Speichern</string>
+    <string name="dialog_website_name_hint">Webseiten Name</string>
+    <string name="dialog_website_page_name_hint">Seitenname</string>
+    <string name="dialog_website_url_hint">Webseiten URL</string>
+    <string name="dialog_website_urlsuffix_hint">(optional) Page number suffix</string>
+    <string name="dialog_website_selector_hint">DOM Selektor</string>
+    <string name="dialog_website_itemperpage_hint">Anzahl der Objekte auf einer Seite</string>
+    <string name="dialog_website_firstpagezero_hint">Wenn die Nummer der ersten Seite 0 ist</string>
+
+    <string name="dialog_websitechooser_title">Wähle deine Favoriten</string>
+
+
+    <!-- About fragment -->
+    <string name="about_title">Über</string>
+    <string name="about_made">Gemacht mit Liebe von</string>
+    <string name="about_made_person" translatable="false">Hugo Gresse http://hugo.gresse.io/</string>
+    <string name="using" translatable="false">Third-party libraries</string>
+    <string name="error_server_unknown" translatable="false">Something went wrong with the server :(</string>
```

Comment:
you can remove the translatable, I'll update the app tomorrow to remove it too on master.